### PR TITLE
Update README.md for javascript_pack_tag

### PR DESCRIPTION
**Pull request by justin808**
Addresses #3035

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,33 @@ You can then link the JavaScript pack in Rails views using the `javascript_pack_
...
+<script src="/packs/map~runtime-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+<script src="/packs/map-16838bab065ae1e314.js" data-turbolinks-track="reload"></script>
+```
+
+**Important:** Pass all your pack names when using this helper otherwise you will
+get duplicated chunks on the page.
+
+```erb
+<%# DO %>
+<%= javascript_packs_with_chunks_tag 'calendar', 'map' %>
```
Comment (wasifhossain) [inline README.md:158]: guess it would be `javascript_pack_tag` rather than `javascript_packs_with_chunks_tag`

File: README.md [hunk 2]
```diff
@@ -135,6 +135,33 @@ You can then link the JavaScript pack in Rails views using the `javascript_pack_
...
+
+<%# DON'T %>
+<%= javascript_packs_with_chunks_tag 'calendar' %>
+<%= javascript_packs_with_chunks_tag 'map' %>
```
Comment (wasifhossain) [inline README.md:162]: same here as above

**Comment by justin808** [comment]
@wasifhossain Thanks for the review. I updated this. @gauravtiwari @jakeNiemiec Can we merge this ASAP?

**Comment by justin808** [comment]
On a related note, we changed the `javascript_packs_with_chunks_tag` to be `javascript_pack_tag`, but maybe we should consider renaming it to `javascript_packs_tag` (packs, not pack) to be clear that this call takes multiple arguments and will generate multiple HTML tags. 

The advantage of this for migration purposes is that many apps will update to v6 and probably miss the need to have all the packs included in one call to the helper which is going to result in horrible performance as the browser will parse the JS multiple times.

**Comment by dhh** [comment]
I think it's better to have poor performance than to break compatibility. But let's stress it loud and clear in the update docs!

**Pull request merged**